client/src/style.css

```diff
@@ -7,7 +7,7 @@
   margin-right: 14px;
 }
 
-@media (min-width: 2160px) {
+@media (min-width: 1790px) {
```

Comment:
These breakpoints don't work for me. The search bar overflows into the itinerary list at around 1790. Changing the breakpoint to 1895 works for me 🤔

Reply:
I've changed it to 1895: https://github.com/opentripplanner/OpenTripPlanner/pull/6091/commits/94943d56379c0b3c479d3fcfc6804ad2810e393a